lib/usd/ui/layerEditor/layerTreeView.cpp

```diff
@@ -136,6 +138,61 @@ void LayerTreeView::onItemDoubleClicked(const QModelIndex& index)
     }
 }
 
+bool LayerTreeView::shouldExpandOrCollapseAll() const
+{
+    return ((QGuiApplication::keyboardModifiers() & Qt::ShiftModifier) != 0);
```

Comment:
if I understand correctly this is a Shift + Click ?

Reply:
Technically this only checks shift but it is a private function called while handling clicks on the expansion button.

Reply:
I think a documentation on Collapse All / Expand All would help to clarify this. Also a plus if the code mentions it.